solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -79,24 +81,27 @@ public abstract class LBSolrClient extends SolrClient {
 
   protected long aliveCheckIntervalMillis =
       TimeUnit.MILLISECONDS.convert(60, TimeUnit.SECONDS); // 1 minute between checks
+  protected int aliveCheckSkipIters = 0;
```

Comment:
Do we use the short hand `Iters` in lots of places?   If not, I like the full `Iteration` as just one less thing to read.

Reply:
maybe just a personal thing though.